tgui/packages/tgui/interfaces/common/LoginScreen.js

```diff
@@ -77,6 +81,15 @@ export const LoginScreen = (_properties, context) => {
               })}
             />
           )}
+          {!!isAdmin && (
```

Comment:
Double `!!` here. Why not just `{isAdmin`

Reply:
Because otherwise this happens:

![image](https://user-images.githubusercontent.com/1496804/94126495-f983dd80-fe57-11ea-934f-94eef60c6b46.png)

Reply:
Can confirm, you MUST do the !! or you get 0s floating around in your interfaces.